starter-core/src/main/java/io/micronaut/starter/feature/function/template/http/httpFunctionJavaController.rocker.raw

```diff
@@ -0,0 +1,49 @@
+@import io.micronaut.starter.application.Project
+
+@args (
+Project project
+)
+
+@if (project.getPackageName() != null) {
+package @project.getPackageName();
+}
+
+
+import io.micronaut.http.annotation.*;
+
+@@Controller("/")
+public class @project.getClassName()Controller {
+
+    @@Get(uri="/", produces="text/plain")
+    public String index() {
+        return "Example Response";
+    }
+
+    @@Post(uri="/", produces="application/json")
```

Comment:
We should use the constants defined in `MediaType` instead of `text/plain` here.

Reply:
Micronaut defaults to json. 

That it is to say: 

Instead   `@Post(uri="/", produces="application/json")`

You can just write: 

`@Post`

I've done the change.